### ISSUE TITLE
Published version of module contain coverage and test folders

### ISSUE THREAD

**Issue by Ginden** [npm/unique-filename#1]
```
$ npm install unique-filename
$ ls node_modules/unique-filename
README.md    README.md~   coverage     index.js     package.json test
```

**Comment by fraxken** [comment]
+1

We are working on a security tool where minified code are flagged (and this one has been flagged because of coverage/prettifiy.js).

it would be even more interesting to manage a whitelist in package.json (instead of .npmignore).

Best Regards,
Thomas

**Comment by lukekarrys** [comment]
Closed by #12
